client-app/shared/static-content/components/breadcrumbs-block.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <VcBreadcrumbs :items="breadcrumbs" />
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import { useBreadcrumbs } from "@/core/composables";
+import { buildBreadcrumbs } from "@/core/utilities";
+
+interface IProps {
+  breadcrumbs: { title: string; seoPath?: string }[];
+}
+
+const props = defineProps<IProps>();
+
+const breadcrumbs = useBreadcrumbs(computed(() => buildBreadcrumbs(props.breadcrumbs)));
```

Comment:
why not 
```ts
const breadcrumbs = useBreadcrumbs(() => buildBreadcrumbs(props.breadcrumbs));
```
or even
```ts
const breadcrumbs = useBreadcrumbs(buildBreadcrumbs(props.breadcrumbs));
```

they should work fine according useBreadcrumbs definition

Reply:
reactivity lost
props.breadcrumbs